src/test/kotlin/com/openlattice/launchpad/LaunchpadSmokeTests.kt

```diff
@@ -25,21 +24,32 @@ class LaunchpadSmokeTests {
     }
 
     companion object {
-        val logger = LoggerFactory.getLogger(LaunchpadSmokeTests::class.java)
         @JvmStatic
-        fun runTestValidateAndCleanup(config: IntegrationConfiguration, vararg sortColumn: String ) {
-            IntegrationRunner.configureOrGetSparkSession( config ).use { session ->
+        fun runArchiveValidateAndCleanup(config: IntegrationConfiguration, vararg sortColumn: String ) {
+            configureOrGetSparkSession( config ).use { session ->
+                ArchiveRunner.runArchives( config, session )
+//                IntegrationValidator.validateIntegration( config, integrationPaths, *sortColumn )
```

Comment:
why is this piece commented out?

Reply:
I haven't written a validator for archives yet, just the current smoke test that runs them.